File: C Programs/Assignment 2/addeven.c
/*Write a program to get  addition of all even numbers and odd numbers between 1 to 10*/


#include<stdio.h>

int main()
{
    int i,even,odd;
    for ( i = 0; i <=10;i++)
     {
         if (i%2==0)
         {
             even=even+i;
           
         }
         else
         {
             odd=odd+i;
         }
     }
       printf("the addition of even numbers between 1 and 10 is %d\n",even);
       printf("addition of odd numbers between 1 and 10 is %d",odd);
}

File: Basic Java/Basic/AddEven.java
package com.cdac.iet;/*Write a Java program to add all even number from 1 to 10 and print it.*/




public class AddEven
{
	public static void main(String args[])
{
	int add=0;
	
	for(int i=0;i<11;i++)
	
	{
		if(i%2==0)
		{
			add=add+i;
		}
		
	}
	System.out.println("Addition of even numbers till 10 is " +add);
}

}